xcode/Subconscious/Shared/Config.swift

```diff
@@ -26,7 +26,11 @@ struct Config: Equatable, Codable {
     /// Only returns test data when Config.debug is enabled
     var fallbackSimulatorQrCodeScanResult: String {
         get {
-            debug ? "did:key:z6MkmCJAZansQ3p1Qwx6wrF4c64yt2rcM8wMrH5Rh7DGb2K7" : ""
+            #if targetEnvironment(simulator)
+            "did:key:z6MkmCJAZansQ3p1Qwx6wrF4c64yt2rcM8wMrH5Rh7DGb2K7"
+            #else
+             ""
+            #endif
```

Comment:
This wasn't actually very useful before since `debug` was always off.

Reply:
Interesting! Makes me wonder if we should define a `private static let isSimulator: Bool` on Config and use that to derive properties like this?

Reply:
Added this!